srcs/Robot.cpp

```diff
@@ -58,17 +69,25 @@ Robot::Robot(simxInt clientID) :
 ** @return: an error or success code as given by V-REP
 */
 simxInt		Robot::doActions(const dna_t &dna) const {
-  simxInt	ret;
-
   for (const auto &action : dna) {
     if (action > highestAction()) {
       cerr << "Robot " << (int)_id << " can't execute " << (int)action << ": It does not exists!" << endl;
       continue;
     }
-    cout << "Robot " << (int)_id << " executing " << actions.at(action)._description << endl;
-    ret = simxSetJointTargetPosition(_clientID, _handlers[action], Robot::actions.at(action)._intensity, simx_opmode_oneshot);
-    if (ret != 0 && ret != 1)
-      return ret;
+    cout << "Robot " << (int)_id << " executing " << _movements.at(action).description << endl;
+    auto movementResult = simxSetJointTargetPosition(
```

Comment:
`auto` is authorized when type is guessable or flagrant.
Here, the third parties library returns an `int` equal to `-1` on errors.